ci/webhook-receiver.py

```diff
@@ -195,13 +216,28 @@ def init_ci_pipeline():
     out, err = proc2.communicate()
 
     if (out):
```

Comment:
I don't think there is any reason to separate this into 2 branches. lets just always do the one that grabs the queue_lock and does w.e processing it needs

Reply:
even with changing it so we check the "out" logic with ci_request_list.length > 0, how can we ignore the else condition? Why would we want to "nuke" that?